input/municipalities.txt

```diff
@@ -0,0 +1,41 @@
+region	municipality	municipality_name	region_name	paakaupunkiseutu	kehyskunta	ymparyskunta
```

Comment:
finnish & english mix-in..? perhaps better to stick with one language, preferrably english..

Reply:
Fixed in https://github.com/HSLdevcom/helmet-data-preprocessing/pull/7/commits/5cb2b56e12b5f3011a632a3b6af582347ffb78d7